engine/src/hopeit/app/api.py

```diff
@@ -96,6 +96,22 @@ def _payload_description(arg: PayloadDef) -> str:
     return str(arg)
 
 
+def _method_summay(title, module) -> str:
+    ds = inspect.getdoc(module)
+    if title is not None:
+        return title
+    if isinstance(ds, str):
+        return ds.split("\n")[0]
+    return ""
+
+
+def _method_description(title, module) -> str:
+    ds = inspect.getdoc(module)
+    if ds is not None and ds.count('\n') > 1:
+        return ds.split("\n", 2)[2]
```

Comment:
Add unit tests for these cases.

Reply:
done on commit https://github.com/hopeit-git/hopeit.engine/pull/23/commits/4db72834126369b5482702b288f3a9d3494ec45d